src/Laravel/Tests/EloquentTest.php

```diff
@@ -87,8 +87,138 @@ public function testDateSearchFilter(): void
             ]
         );
 
-        $response = $this->get('/api/books?publicationDate='.$updated['publicationDate'], ['accept' => ['application/ld+json']]);
-        $this->assertSame($response->json()['member'][0]['@id'], $book['@id']);
+        $response = $this->get('/api/books?publicationDate[eq]='.$updated['publicationDate'], ['accept' => ['application/ld+json']]);
+        $this->assertSame($response->json()['hydra:member'][0]['@id'], $book['@id']);
+    }
+
+    public function testDateFilterIncludeNull(): void
+    {
+        $response = $this->get('/api/books', ['accept' => ['application/ld+json']]);
+        $book = $response->json()['hydra:member'][0];
+        $updated = $this->patchJson(
+            $book['@id'],
+            ['publicationDate' => null],
+            [
+                'accept' => ['application/ld+json'],
```

Comment:
Nit:
```suggestion
                'Accept' => ['application/ld+json'],
```

Same everywhere.